lib/honeybadger/config/defaults.rb

```diff
@@ -287,6 +288,21 @@ class Boolean; end
         description: 'Send exceptions when retrying job.',
         default: true,
         type: Boolean
+      },
+      :'breadcrumbs.enabled' => {
+        description: 'Enable/Disable breadcrumb functionality.',
+        default: true,
+        type: Boolean
+      },
+      :'breadcrumbs.active_support_notifications' => {
```

Comment:
Since this is Ruby-only config, I'm thinking that we may want to make this a method on `Honeybadger::Config::Ruby` rather have it in the defaults (which is also used to build values in the yaml and env config styles). For example, currently if you set the environment variable `HONEYBADGER_BREADCRUMBS_ACTIVE_SUPPORT_NOTIFICATIONS=foo` (lol), this value would result in the string `"foo"`.

Reply:
Then again, the default value also has to get set somewhere. So don't mistakenly assume I've thought this all through. :)

Reply:
Well since they can only replace with a string via ENV var, think it would be worth checking for string type and disabling if we find it?

Reply:
Here's an idea: you could create a new "type" called `RubyOnly` (or something), [similar to `Boolean`](https://github.com/honeybadger-io/honeybadger-ruby/blob/4b401e83a93bc243cce28c2ba4053985807fa5f7/lib/honeybadger/config/defaults.rb#L6), and then just drop those options in `Config::Env`:

https://github.com/honeybadger-io/honeybadger-ruby/blob/4b401e83a93bc243cce28c2ba4053985807fa5f7/lib/honeybadger/config/env.rb#L14

Reply:
Humm, that's an interesting option.

How about I make it more complicated?

Since I added "select_keys" option, they could very well make a valid custom config in the YAML format as well.

Reply:
That works for me, too. It sounds we just need a way to make `Config::Env` skip options. I still kind of like the type idea, but I'll defer to whatever implementation you think works best.

Reply:
Okay what do you think about the ability to ignore specific types in the ENV class?

https://github.com/honeybadger-io/honeybadger-ruby/blob/e5a89f59a054e8ac72cf4652a8e83a4364012e9a/lib/honeybadger/config/env.rb#L9-L23